plugins/renepay/mcf.c

```diff
@@ -1170,7 +1166,7 @@ static struct flow **
 			}
 
 
-			struct flow *fp = tal(list_ctx,struct flow);
+			struct flow *fp = tal(this_ctx,struct flow);
```

Comment:
tallocate this off flows...

Reply:
Caveat: these are flow candidates, `minflow` produces different sets flows and selects one. Ownership of the flows to the flow array is elegant and allows me to do:
```
if(is_better(candidate_flow_array,best_flow_array))
{
    tmp = best_flow_array;
    best_flow_array = candidate_flow_array;
    tal_free(tmp);
}
```